### PR TITLE
Fix nodecmd tests

### PR DESCRIPTION
**Pull request by blukat29**
## Proposed changes

- Fix nodecmd tests related to --datadir flags.

nodecmd tests are not run by Pull-Request CI. It only runs as part of nightly-coverage CI.
To run manually,
```
cd cmd/utils/nodecmd && go test -v
```

## Types of changes

- [x] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

### CODE REVIEW

**Review by kjeom — APPROVED** [review]

File: cmd/utils/nodecmd/accountcmd_test.go [hunk 1]
```diff
@@ -46,7 +46,8 @@ func tmpDatadirWithKeystore(t *testing.T) string {
 }
 
 func TestAccountListEmpty(t *testing.T) {
-	klay := runKlay(t, "klay-test", "account", "list")
+	datadir := tmpdir(t)
+	klay := runKlay(t, "klay-test", "account", "list", "--datadir", datadir)
```
Comment (yoomee1313) [inline cmd/utils/nodecmd/accountcmd_test.go:50]: I think datadir doesn't need to be set here. The datadir is automatically set in runKlay func, right?

File: cmd/utils/nodecmd/consolecmd_test.go [hunk 2]
```diff
@@ -53,15 +53,15 @@ func TestConsoleWelcome(t *testing.T) {
...
 	klay.SetTemplateFunc("apis", func() string { return ipcAPIs })
+	klay.SetTemplateFunc("datadir", func() string { return klay.Datadir })
 
 	// Verify the actual welcome message to the required template
 	klay.Expect(`
 Welcome to the Klaytn JavaScript console!
 
 instance: Klaytn/{{klayver}}/{{goos}}-{{goarch}}/{{gover}}
- datadir: {{.Datadir}}
+ datadir: {{datadir}}
```
Comment (yoomee1313) [inline cmd/utils/nodecmd/consolecmd_test.go:64]: The original code is still working well. Why don't you use `.Datadir`?

File: cmd/utils/nodecmd/accountcmd_test.go (hunk 1 above)
Comment (blukat29) [reply cmd/utils/nodecmd/accountcmd_test.go]: You're right. It got fixed by MigrateGlobalFlags. Reverted to original at 1929acc1.

File: cmd/utils/nodecmd/consolecmd_test.go (hunk 2 above)
Comment (blukat29) [reply cmd/utils/nodecmd/consolecmd_test.go:64]: Just cosmetic reason, to make it consistent with `testAttachWelcome`.

**Review by yoomee1313 — APPROVED** [review]

**Review by JayChoi1736 — APPROVED** [review]

**Review by kjeom — APPROVED** [review]

**Review by aidan-kwon — APPROVED** [review]

**Pull request merged**